styles/config.css

```diff
@@ -342,3 +348,9 @@ footer li {
 footer a:hover {
   text-decoration: none;
 }
+
+@media only screen and (max-width: 768px) {
+ .social-links ul {
+    width: 325px;

```

Comment:
@karinisantanna we should center align the social buttons here. Mind making necessary changes?

Reply:
@cTxplorer Hi! yes, i revert this part! Can you validate please?